src/lib/IFrameComm.js

```diff
@@ -0,0 +1,201 @@
+import React, { Component } from 'react';
```

Comment:
This is code copied from `react-iframe-comm`? We probably need to preserve the upstream license information in this file (and any others we're copying over.).